src/compositor/view.h

```diff
@@ -59,6 +59,10 @@ struct wlc_view {
    } state;
 };
 
+static inline bool is_x11_view(struct wlc_view *view) {
```

Comment:
Put storage class and return value in own line e.g:
```
static inline bool
is_x11_view(...)
{
   ...
}
```

This is the style wlc uses.